installer/tue-install-impl.bash

```diff
@@ -879,16 +843,7 @@ done
 
 
 # (Re-)generate setup file
-mkdir -p "$TUE_ENV_DIR"/.env/setup
-echo "# This file was auto-generated by tue-install. Do not change this file." > "$TUE_ENV_DIR"/.env/setup/target_setup.bash
-
-mkdir -p "$TUE_INSTALL_DEPENDENCIES_DIR"
-installed_targets=$(ls "$TUE_INSTALL_DEPENDENCIES_DIR")
-TUE_SETUP_TARGETS=" "
-for t in $installed_targets
-do
-    generate_setup_file "$t"
-done
+_generate_setup_file
```

Comment:
I think this can be moved completely to tue-functions.bash as after tue-install-impl exits, the stack goes back to `tue-get`

Reply:
Do we want it happen before or after all the installs?

Reply:
I would say after. Then the setup file would be the right one. And if install/update fails then the setup file doesn't change.